### ISSUE TITLE
[RFC] Should we implement smartcrop.js

### ISSUE THREAD

**Issue by I-Valchev** [bolt/core#2818]
https://github.com/jwagner/smartcrop.js

**Comment by Dsduit** [comment]
I think so. Would be a nice feauture that is requested regularly by clients

**Comment by peterboorsma** [comment]
I love this but IMHO this should be an extension.

**Comment by Puddingboy** [comment]
I agree with @peterboorsma for what it's worth. I think this should not be a matter of on/off though. I think individual fields should be able to enable or disable this. In my case it would be highly recommended and desired for product images, but highly undesirable for background images. I would also strongly prefer to be able to enable/disable this slightly before the image upload and processing starts in the record backend and in the filemanager.

I think a great starting point would be a switch next to a common image and imagelist fieldtype, before trying to extend contenttype features.

I know this is already very deep into the implementation part, while you guys are still discussing aye/nye, but I felt this would be the sensible way of implementing this type of feature without creating conflict.
